### PR TITLE
fix(style): set line-height to improve global body legibility

### LINKED ISSUE #591: Improve legibility of body copy by increasing line height
Currently our line height is equal to the font size.
screen shot 2018-10-10 at 3 14 12 pm

Using the [Golden Ratio Typography Calculator](https://grtcalculator.com) we can see that for content of this width, the line height should be 23px.
screen shot 2018-10-10 at 3 14 06 pm

Much better, no?

Would love to see this applied to our terms of service, and privacy policy (24px line height). Are there other areas with paragraphs of copy we can improve?

### PR DESCRIPTION
**Pull request by karansapolia**
fixes #591

continuation of https://github.com/mozilla/fxa-content-server/pull/7079

### CODE REVIEW

**Comment by karansapolia** [comment]
@ryanfeeley review please

**Comment by karansapolia** [comment]
> One final place where it might make sense is the update firefox template.

>Can you make the selectors specific for now, we can try to make things more general in the future, but that's not a task I think you should take on here because it's could be a doozy ensuring everything looks just as it should.

Done. @shane-tomlinson r+? Would also request you to rerun the tests.

File: packages/fxa-content-server/app/styles/modules/_update_firefox.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#fxa-cookies-disabled {
```
Comment (shane-tomlinson) [inline packages/fxa-content-server/app/styles/modules/_update_firefox.scss]: Down below you use the id `fxa-update-firefox` and here you are using `fxa-cookies-disabled`. Expected?

Comment (karansapolia) [reply packages/fxa-content-server/app/styles/modules/_update_firefox.scss]: No, Corrected it.

**Pull request merged**